tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -879,12 +879,31 @@ def _VerifyGraphDefV2(self, run_params, original_gdef, gdef_to_verify,
       # 2.0, but we still make sure that the converted ops are gone from the
       # graph.
       unexpected_names = set(nest.flatten(expected_engines.values()))
-      self.assertEmpty(
-          [name for name in unexpected_names if name in all_op_names])
-      expected_engines = set(expected_engines.keys())
-
-    self.assertEqual(set(expected_engines), trt_op_names)
-
+      try:
+        self.assertEmpty(
+            [name for name in unexpected_names if name in all_op_names])
+      except AssertionError:
+        raise RuntimeError("""Unexpected TF-TRT Engines returned:,
+All Op Names: {},
+TRT Op Names: {},
+Expected: {}""".format(
+          all_op_names,
+          trt_op_names,
+          unexpected_names,
+        ))
+
+    expected_engines = set(expected_engines.keys())
+    try:
+      self.assertEqual(expected_engines, trt_op_names)
+    except AssertionError:
+      raise RuntimeError("""Unexpected TF-TRT Engines returned:,
+        All Op Names: {},
+        TRT Op Names: {},
+        Expected Engines: {}""".format(
+          all_op_names,
+          trt_op_names,
+          expected_engines,
```

Comment:
I am getting this error with I run and the tests with --test_env=TF2_BEHAVIOR=1:


 File "/build/work/fa3a0e10bc497b211fc2e1e6537b38cd4e12/google3/runfiles/google3/third_party/tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py", line 895, in _VerifyGraphDefV2
    expected_engines = set(expected_engines.keys())
AttributeError: 'list' object has no attribute 'keys'

Reply:
Oh I see the issue, let me fix that one ;)